src/config/dynamic-theme-fixes.config

```diff
@@ -1392,6 +1392,21 @@ table#calendar td {
 
 ================================
 
+cynkra.com
+
+INVERT
+.btn-outline-dark
+.shadow
+.navbar-dark .navbar-brand
+.cynkra-logo-font
+
+CSS
```

Comment:
Make sure it follows the code-style and look like
```CSS
.btn-outline-secondary {
    color: var(--darkreader-neutral-text) !important;
}
```

Reply:
@Gusted Thanks, done.